### DISCUSSION TITLE
IoT Support: MQTT Integration possible/planned ?

### DISCUSSION THREAD

**Discussion by MeisterAdebar** [revoxhere/duino-coin#1438] [Ideas]
I think the new IoT support should at least include MQTT integration.

What do I get from the measured value of a sensor in my wallet ?

Why reinvent the world, is it possible to integrate the code (21KB) from Duino-Coin into Theo Arends' Tasmota firmware?

**Comment by revoxhere** [comment]
ESP32 code includes MQTT support also.

**Reply by MeisterAdebar** [reply]
Ah, ok thanks, I only looked at the ESP8266 code (there is no MQTT integration there yet)

**Comment by MeisterAdebar** [comment]
If the integration in Tasmota is not desired, maybe someone can answer the question of whether the integration would be possible in principle and, if so, show an example (I'm an absolute layman in Arduino programming) how to approach it.

For example, I implemented an IR transmitter based on Tasmota, the ESP basically just waits all day to turn on the television, usually once in the evening.
In my opinion, it would make sense to give the ESP a job (Duino-Coin) and not just use electricity all day.

**Comment by bigjohnmax** [comment]
I have 8 esp01 integrated in homeassistant, it would be very useful if they could mine and trigger the relays.
